starknet-core/src/types/transaction_receipt.rs

```diff
@@ -24,6 +24,9 @@ pub struct Receipt {
     #[serde_as(as = "UfeHex")]
     pub transaction_hash: FieldElement,
     pub transaction_index: Option<u64>,
+    #[serde(default)]
+    #[serde_as(as = "Option<UfeHex>")]
+    pub actual_fee: Option<FieldElement>,
```

Comment:
The `actual_fee` field also appears in a [block](https://alpha4.starknet.io/feeder_gateway/get_block?blockId=PENDING) under `transaction_receipts`, but it's missing in our `ConfirmedReceipt` object.

Reply:
I actually checked before excluding... I think it's the same case as `class_hash`. Only new blocks have it I guess.

Let me double check and create separate test data for these stuff.

Reply:
I added assertions that old blocks don't have these fields so that we won't miss them fixing it, if ever.